### ISSUE TITLE
Using a different "consistency level" for a ColumnFamily.

### ISSUE THREAD

**Issue by msfbrasil** [Impetus/kundera#588]
Hello there!

Is there any way to configure the consistency level just for one entity?

We are using Kundera version 2.11.
Our original code creates the EntityManagerFactory passing the map with customized properties, including the consistence level: ConsistencyLevel.ONE.

To access the mentioned entity using ConsistencyLevel.QUORUM, I've tried to set this configuration after retrieving the EntityManager on our DAO as shown below:
# 

EntityManager entityManager = this.entityManagerProvider.get();
entityManager.setProperty("consistency.level", consistencyLevel);
# 

Obs.: We use guice injection to build the EntityManagerFactory and EntityManager.

The problem is that informing or not the "ConsistencyLevel.ONE" during "EntityManagerFactory" instantiation, this results on a Thrift client instance that uses this specific consistency level by default.

It's kind the "setProperty" above has no effect.

Debugging Kundera's code, is possible to verify that the value of "consistencyLevel" member of "CassandraClientBase" is "ONE" on line below:
# 

return (T) conn.execute_cql3_query(ByteBuffer.wrap(query.getBytes(Constants.CHARSET_UTF8)), Compression.NONE, consistencyLevel);
# 

Am I doing something wrong?

Thanks and best regards,
Mauro.

**Comment by msfbrasil** [comment]
Problem solved!

Most of posts regarding the subject presented examples where the ConsistenceLevel were provided as a enum (like, ConsistenceLevel.ONE).

Auto-complete always give me two options, being: 1- "org.apache.cassandra.db.ConsistencyLevel", and 2- "org.apache.cassandra.thrift.ConsistencyLevel".

Minding the independence of client implementation, I've used the prior option, that didn't work because it was not a type recognized by "Thrift" client implementation.

Digging the code, I've noticed that Kundera's internal code converts the "String" value to respective enum value.

So, correct me if I'm wrong, "String" type (with values: "QUORUM", "ONE", "ANY", etc) is the best approach to configure the consistency level.

I've changed everything to use the consistency level as "String" and everything is working fine now.

Thanks and best regards,
Mauro.
